manywheel/build_docker.sh

```diff
@@ -40,42 +40,59 @@ case ${GPU_ARCH_TYPE} in
         ;;
 esac
 
-DOCKER_IMAGE=${DOCKER_REGISTRY}/pytorch/manylinux-builder:${DOCKER_TAG}
-(
-    set -x
-    DOCKER_BUILDKIT=1 docker build \
-        -t "${DOCKER_IMAGE}" \
-        ${DOCKER_GPU_BUILD_ARG} \
-        --build-arg "GPU_IMAGE=${GPU_IMAGE}" \
-        --target "${TARGET}" \
-        -f "${TOPDIR}/manywheel/Dockerfile" \
-        "${TOPDIR}"
-)
+IMAGES=''
+for DOCKER_NAME in manylinux manylinux2014; do
```

Comment:
I think if we're going to do this it should just be an env variable that you pass through instead of iterating a `for` loop here, something like:

```
MANYLINUX_VERSION=2014 manywheel/build_docker.sh
```

Just so that we don't have to build all of the variants every single time.

This logic to build all of the variants can be added to `manywheel/build_all_docker.sh`

Reply:
adopted